src/main/kotlin/com/exactpro/th2/check1/entities/TaskTimeout.kt

```diff
@@ -0,0 +1,5 @@
+package com.exactpro.th2.check1.entities
+
+data class TaskTimeout(val messageTimeout: Long? = null, val timeout: Long) {
```

Comment:
If you want to use the default values you should put the parameters without ones first